config/config.exs

```diff
@@ -5,4 +5,4 @@ use Mix.Config
       {"localhost", 9093},
       {"localhost", 9094},
     ],
-    consumer_group: false
+    consumer_group: "kafka_ex_dev"
```

Comment:
Is there a way to test that support?  From what I can tell, the consumer group setting here was being inconsistently honored throughout the code.  In some places, the default value of "kafka_ex" was being used.